src/encoded/static/components/item-pages/HealthView.js

```diff
@@ -167,16 +172,23 @@ export default class HealthView extends React.PureComponent {
     }
 
     render() {
-        const { context, schemas, session, windowWidth, href, keyTitleDescriptionMapConfig, keyTitleDescriptionMapCounts, excludedKeys } = this.props;
+        const { context: propContext, schemas, session, windowWidth, href, keyTitleDescriptionMapConfig, keyTitleDescriptionMapCounts, excludedKeys } = this.props;
```

Comment:
Codacy found an issue: ['context' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6028006)